ReactWindows/ReactNative/DevSupport/DevSupportManager.cs

```diff
@@ -121,7 +130,12 @@ public void HandleException(Exception exception)
             var progressDialog = new ProgressDialog("Please wait...", "Fetching JavaScript bundle.");
             var dialogOperation = progressDialog.ShowAsync();
 
-            if (_jsBundleFile == null)
+            if (_isUsingJsProxy)
+            {
+                await ReloadJavaScriptInProxyMode(progressDialog.Token);
```

Comment:
Determine why progress dialog can end prematurely (how is the async context being lost?)